tests/TestCase/TestSuite/Fixture/DeleteQueryStrategyTest.php

```diff
@@ -41,10 +41,9 @@ public function testTeardownTest()
     {
         $articleIndex = IndexRegistry::get('Articles');
 
-        $strategy = new DeleteQueryStrategy($this->fixtureManager);
         $this->assertCount(2, $articleIndex->find()->all());
 
-        $strategy->teardownTest();
+        (new DeleteQueryStrategy())->teardownTest();
```

Comment:
Could this test be failing because `$this->fixtures` is empty because setupTest() was not called?